From 497c53902a4e1276e1406eecb0049ca153aa9cb9 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Fri, 26 Oct 2018 11:03:22 +0200
Subject: [PATCH] tqma8qx: mba8qx: dt fixes for usb otg1

do not set dr_mode in dt, must be handled in board code

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 arch/arm/dts/fsl-imx8qxp-tqma8qx-mba8qx.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/dts/fsl-imx8qxp-tqma8qx-mba8qx.dts b/arch/arm/dts/fsl-imx8qxp-tqma8qx-mba8qx.dts
index 7319fa2339..cbc6179a66 100644
--- a/arch/arm/dts/fsl-imx8qxp-tqma8qx-mba8qx.dts
+++ b/arch/arm/dts/fsl-imx8qxp-tqma8qx-mba8qx.dts
@@ -253,7 +253,7 @@
 	adp-disable;
 	power-polarity-active-high;
 	disable-over-current;
-	dr_mode = "host";
+/*	dr_mode = "host"; */
 	status = "okay";
 };
 
